### PR TITLE
Add Earthly to Monorepo Tools

### PR DESCRIPTION
**Pull request by brandonSc**
Hi folks, 👋 

Really enjoyed the Monorepo.tools webpage. 

[Earthly](https://github.com/earthly/earthly) is another great build tool when working in a monorepo. Its built-in caching can speed up development since it only rebuilds sub-components of a monorepo that are affected by a change.

Earthly also has a sophisticated Dockerfile-like syntax that lets you script just about anything in terms of build steps, unit or integration tests, code analysis, etc. Everything runs in sandboxed containers too, so something like a build failure in CI can be easily reproduced and fixed on a dev machine.

Some examples of Earthly in a monorepo can be found [here](https://github.com/earthly/earthly/tree/main/examples/monorepo) and [here](https://github.com/earthly/earthly/tree/main/examples/go-monorepo).

I hope you'll find this a welcome addition to the webpage. Our fast-growing community of Earthly users actually suggested that we should be listed here.

### CODE REVIEW

File: libs/website/ui-home/src/lib/tools-review.tsx [hunk 1]
```diff
@@ -25,6 +25,12 @@ const tools = [
     description:
       'A fast, scalable, multi-language and extensible build system.',
   },
+  {
+    title: 'Earthly',
+    organization: 'Earthly Technologies',
+    organizationUrl: 'https://earthly.dev/?utm_source=monorepo.tools',
+    description: 'The effortless CI/CD framework that runs anywhere.',
+  },
```
Comment (brandonSc) [inline libs/website/ui-home/src/lib/tools-review.tsx:33]: There are some [`Fake cards backgrounds`](https://github.com/nrwl/monorepo.tools/blob/main/libs/website/ui-home/src/lib/tools-review.tsx#L527-L554) on this part of page that should be adjusted to accommodate the extra column for Earthly, but I didn't understand what formula to calculate the correct widths. 

If someone can point me in the right direction I can fix that up.

**Comment by bcabanes** [comment]
Hi @brandonSc , thank you for your time and contribution. We are reviewing the application since we did not know of Earthly before.

**Comment by brandonSc** [comment]
> Hi @brandonSc , thank you for your time and contribution. We are reviewing the application since we did not know of Earthly before.

Thanks @bcabanes - and happy to answer any questions you have about Earthly.

**Comment by bcabanes** [comment]
Hi, thank you for your time! 
From the information, we gathered and the usages explained on https://earthly.dev/. We prefer to not add Earthly to monorepo.tools since it seems to be primarily a build system for containers and is not addressing specifically the different monorepo problems.

Earthly is used as a generic CI abstraction, both in monorepos and polyrepos situations. It does not focus specifically on helping developers with the main monorepo issues they face while developing their applications or packages.

We will probably add another section on the website for "related tools", we feel Earthly would be a good choice there.

**Comment by vladaionescu** [comment]
> primarily a build system for containers
> and is not addressing specifically the different monorepo problems

It happens to use containers underneath - but that's simply an implementation detail. In fact, it addresses all the problems highlighted in this change - which are very much monorepo-specific problems.

Can you be more specific about how you think that Earthly is not addressing the monorepo problems indicated as part of this PR? (Brandon and I would be happy to join a zoom call if it's easier to discuss live)

**Comment by tclasen** [comment]
As an Earthly user, I'd like to see it added to the comparison list of other great monorepo tooling. One of the things Earthy does better than anything else on this list is isolation and caching of tasks and builds by leveraging the excellent OCI ecosystem. This means not only can all tasks be remotely executed and cached, but even for local tasks I don't need to have the tooling installed to be able to run that task locally, since it is all done in an isolated environment.

**Comment by jmgilman** [comment]
> We prefer to not add Earthly to monorepo.tools since it seems to be primarily a build system for containers and is not addressing specifically the different monorepo problems.

This seems to be misinformed. A single example: the feature of sharing Earthfiles via `FROM ../project+target` is a huge boon for monorepos.

**Pull request closed without merging**